### ISSUE TITLE
Link not functional for keyboards or screenreaders

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#88961]
## Summary
On canceled appointments, the link to Schedule a new appointment isn't accessible via keyboard or screenreader.

<img width="1618" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemY4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c6b0e2068fa677635a40414ae88ce6a9497afd4b/Screenshot%202024-07-22%20at%201.29.31%E2%80%AFPM.png" alt="Screenshot 2024-07-22 at 1.29.31 PM.png" />

<img width="886" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeno4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4ff7b12a689cfab13f834be590d76d14b072b8a1/Screenshot%202024-07-22%20at%201.30.45%E2%80%AFPM.png" alt="Screenshot 2024-07-22 at 1.30.45 PM.png" />

[Video walkthrough of issue](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMEw4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6651868754cd3924254ac148562b88f510329851/Screen%20Recording%202024-07-22%20at%201.26.04%E2%80%AFPM.mov)

## Specs
- Device: MacBook Pro
- Browser: Chrome

## Steps to Reproduce
**Test User:** royal.tester@id.me

 1.  https://staging.va.gov/my-health/appointments/
 2. Choose any canceled appointment
 3. Tab to the link to "Schedule a new appointment"
 4. Use ~~Space or~~ (note: I misspoke in the video, the space key doesn't activate links) Enter to try to navigate to the link via keyboard. This did not work for me, though clicking the link did (see video)
 5. Turn on voiceover and repeat the steps above.
 6. Voiceover announces the link as a group, and can't navigate to it.
 

### Actual Result
_Describe what happened that was unexpected or undesired; including screenshots is encouraged_

The link was not accessible via keyboard and was not announced as a link to screenreader users.

### Desired Result
_Describe what was supposed to happen; reference existing Github issue that contains a solution if possible_

The link should be accessible via keyboard and announce as a link to screenreader users.

## Definition of Done
- [x] Desired result is achieved and/or document decisions made
- [ ] Relevant documentation is updated

**Comment by Bren22va** [comment]
passing off to @outerpress

**Comment by outerpress** [comment]
This is working as expected, thank you! Fyi @Bren22va @ldelacosta
